.github/workflows/ci.yml

```diff
@@ -28,6 +28,14 @@ jobs:
             dep ensure
         fi
 
+    - name: Get kubebuilder
+      run: |
+        os=$(go env GOOS)
+        arch=$(go env GOARCH)
+        curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
```

Comment:
Please check the sha256 of the file you download.

Reply:
Done